_data/accessibility-tests/select.yml

```diff
@@ -0,0 +1,113 @@
+    wcag_criterion: 2.4.7
+  - summary: Navigating to select doesn't trigger a change of context.
+    summary_additional: |
+      When you type into the select,
+      the context of the select component doesn't change.
+      For example, landing on select doesn't cause a window to open or focus to jump to another field.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.1
+# Screen reader tests
+  - summary: Screen reader announces select dropdown items in a list.
+    summary_additional: |
+      When you navigate with arrow keys,
+      `enter` and `spacebar` key while using a screen reader, you hear the the drop down label and items in the list.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.1
+  - summary: Select options are read in the same order that they appear on the page.
+    summary_additional: |
+      When you use a screen reader and navigate through the dropdown list,
+      you hear each text option in the same order in which they appear on the page.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.2
+  - summary: Screen reader provides information about the purpose of the select.
+    summary_additional: |
+      When you are using a screen reader,
+      you will know the common purpose of the select option (e.g., "Select your state").
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 1.3.5
+  - summary: Screen reader announces user is in a select element.
+    summary_additional: |
+      When you use a screen reader and access the select component,
+      you will hear that you have entered a "combo box", "select",  or "menu pop-up" element,
+      and that you can select an option.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.6
+  - summary: Content doesn't change until the user takes an action to change it.
+    summary_additional: |
+      When using a screen reader to navigate to the select,
+      if you don't take a prompted action, the content stays the same.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.2
+  - summary: Select components are identified consistently.
+    summary_additional: |
+      When using a screen reader to navigate a page with multiple select components,
+      each of them is announced as a select.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.2.4
+  - summary: Screen reader announces all instructions.
+    summary_additional: |
+      When you use a screen reader and navigate to a select option,
+      you hear every instruction for how to operate and for any options available.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.0
+    wcag_criterion: 3.3.2
```

Comment:
In other tests, we've changed this criterion to a be related to `content` since it only states that the input should have an input. Criterion `1.3.1` handles the screen reader label callouts.

Reply:
@amycole501 @alex-hull Do you have a preference on this? Is it ok to move this to the general section? If so, we'll need to re-write a bit

Reply:
Yes let's move it to general thanks!

Reply:
I moved this to general and updated the summary to the following in [29afe9b](https://github.com/uswds/uswds-site/pull/2702/commits/29afe9b347a90e5678a2ad32bcc3194f35350e13): 

> **Select component provides labels or instructions.** When you view or interact with the select component, you see or hear a label that describes the expected content for the component and the options available.